src/TTabBar.cpp

```diff
@@ -33,14 +33,14 @@
 void TStyle::drawControl(ControlElement element, const QStyleOption *option, QPainter *painter, const QWidget *widget) const
 {
     if (element == QStyle::CE_TabBarTab) {
-        QString tabText = mpTabBar->tabText(mpTabBar->tabAt(option->rect.center()));
+        QString tabString = mpTabBar->tabData(mpTabBar->tabAt(option->rect.center())).toString();
```

Comment:
```suggestion
        QString tabName = mpTabBar->tabData(mpTabBar->tabAt(option->rect.center())).toString();
```

Reply:
Trying out this new suggestions feature, and I think `tabName` is a more appropriate name here.

Reply:
I wanted to change away from using `tabText` because that is *also* the property (and the getter method) and using the same name as a variable name can sometimes trip a *human* code reader up...!

Reply:
So, let us see what happens when I hit **Apply suggestion** :smiling_imp: ...